lib/concurrent-ruby/concurrent/atomic/locals.rb

```diff
@@ -146,23 +139,23 @@ def locals!
   # An array-backed storage of indexed variables per thread.
   class ThreadLocals < AbstractLocals
     def locals
-      Thread.current.thread_variable_get(@name)
+      Thread.current.thread_variable_get(:concurrent_thread_locals)
     end
 
     def locals!
       thread = Thread.current
-      locals = thread.thread_variable_get(@name)
+      locals = thread.thread_variable_get(:concurrent_thread_locals)
```

Comment:
This change means that multiple instances of the class will clobber each other.

Reply:
Yes but since we only need 1 instance we support just that and not more

Reply:
Okay, fair enough.